spec/features/admin/dashboard_spec.rb

```diff
@@ -1,15 +1,49 @@
 require "rails_helper"
 
 describe "admin dashboard" do
+  let!(:admin) { create :admin }
+
+  context "authorization" do
+    xit 'only admins can see the admin dashboard' do
+      user  = create :user
+      allow_any_instance_of(ApplicationController).to receive(:current_user).
+        and_return(user)
+      visit admin_dashboard_path(user)
+      expect(current_path).to eq(root_path)
+      allow_any_instance_of(ApplicationController).to receive(:current_user).
+        and_return(admin)
+      visit admin_dashboard_path(admin)
+      expect(current_path).to eq(admin_dashboard_path(admin))
+    end
+  end
+
   context "new business approval" do
-    let!(:admin) { create :admin }
+    let!(:business) { create :business }
 
     it "displays requests" do
-      Business.create(user: admin, name: "New Water Merchant")
       allow_any_instance_of(ApplicationController).to receive(:current_user).
         and_return(admin)
       visit admin_dashboard_path(admin)
-      expect(page).to have_content("Water Merchant")
+      expect(page).to have_content("Water merchant")
+    end
+
+    it 'approve request' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.